docs/CLI_Reference/oras_repo_tags.md

```diff
@@ -0,0 +1,70 @@
+## oras repo tags
+
+Show tags of the target repository.
+
+```bash
+oras repo tags [flags] <name>
+```
+
+### Synopsis
+
+Show tags of the target repository:
+
+```bash
+oras repo  tags localhost:5000/hello
+```
+
+Show tags in the target repository with digest-like tags hidden:
+
+```bash
+oras repo  tags --exclude-digest-tag localhost:5000/hello
+```
+
+Show tags of the target repository that include values lexically after last:
+
+```bash
+oras repo  tags --last "last_tag" localhost:5000/hello
+```
+
+Show tags of the target OCI layout folder 'layout-dir':
+
+```bash
+oras repo  tags --oci-layout layout-dir
+```
+
+Show tags of the target OCI layout archive 'layout.tar':
+
+```bash
+oras repo  tags --oci-layout layout.tar
+```
+
+Show tags associated with a particular tagged resource:
+
+```bash
+oras repo  tags localhost:5000/hello:latest
+```
+
+Show tags associated with a digest:
+
+```bash
+oras repo  tags localhost:5000/hello@sha256:c551125a624189cece9135981621f3f3144564ddabe14b523507bf74c2281d9b
+```
+
+## Options
+
+```
+      --ca-file string              server certificate authority file for the remote registry
+  -d, --debug                       debug mode
+      --exclude-digest-tags         exclude all digest-like tags such as 'sha256-aaaa...'
+  -h, --help                        help for tags
+      --insecure                    allow connections to SSL registry without certs
+      --last last                   start after the tag specified by last
+      --oci-layout                  Set target as an OCI image layout.
```

Comment:
This option is a capitalized and with a period at the end for some reason. I noticed it is like that for other commands as well.

Reply:
Fixed